addons/udes_delivery_control/views/stock_picking.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_form_stock_picking" model="ir.ui.view">
+        <field name="name">udes_delivery_control_form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <!-- Disable tabs on Delivery Control Picking -->
+            <!-- Disable Detailed Operations tab -->
+            <xpath expr="//page[1]" position="attributes">
+                <attribute name="attrs">{'invisible': ['|', ('show_operations', '=', False), ('u_is_delivery_control', '=', True)]}</attribute>
+            </xpath>
+            <!-- Disable Operations tab -->
+            <xpath expr="//page[2]" position="attributes">
+                <attribute name="attrs">{'invisible': ['|', ('show_operations', '=', False), ('u_is_delivery_control', '=', True)]}</attribute>
+            </xpath>
+            <!-- Make partner field mandatory -->
+            <xpath expr="//field[@name='partner_id']" position="attributes">
+                <attribute name="attrs">{'required': [('u_is_delivery_control', '=', True)]}</attribute>
+            </xpath>
+            <!-- Add page to notebook -->
+            <xpath expr="//notebook[1]" position="inside">
+                <page string="Door Control" name="door_control" attrs="{'invisible': [('u_is_delivery_control', '=', False)]}">
+                    <field name="u_extras_id" invisible="1"/>
+                    <field name="u_is_delivery_control" invisible="1"/>
+                    <group colspan="4">
+                        <field name="u_is_backload" string="Backload ?"/>
```

Comment:
Nit: Could we change the string to `Backload Required?` - just makes it a bit more clearer I think

Reply:
done